sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/SyncActivity.kt

```diff
@@ -126,26 +134,31 @@ class SyncActivity : DuckDuckGoActivity() {
             .launchIn(lifecycleScope)
 
         viewModel.commands().flowWithLifecycle(lifecycle, Lifecycle.State.CREATED).onEach { processCommand(it) }.launchIn(lifecycleScope)
+        viewModel.observeState()
```

Comment:
I did want to avoid doing this we because it feels wrong to call `viewModel.viewState()` and also need to call `viewModel.observeState()`. They seem duplicated. Is this something we do in other parts of the app?